src/components/DialogBody.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import CloseIcon from './CloseIcon';
+
+const DialogBody = ({ locationDetails, onClick }) => (
+  <>
+    <button
+      className="dialog__button"
+      aria-label="close dialog"
+      onClick={onClick}
+    >
+      <CloseIcon />
+    </button>
+
+    <div className="dialog__image">
+      <img
+        src={
+          locationDetails?.thumbnail?.source ||
+          `${process.env.PUBLIC_URL}/placeholder.png`
+        }
+        alt={locationDetails?.thumbnail?.source ? locationDetails.title : ''}
+      />
+    </div>
+    <div className="dialog__body">
+      <a
+        className="dialog__body__link"
+        href={`https://en.wikipedia.org/?curid=${locationDetails.pageid}`}
+      >
+        <h2>{locationDetails.title}</h2>
```

Comment:
On another hand, I believe that when you use ':hover' event over an element in mobile devices (in this case a link), it is 'not' totally helpful, because it's difficult to simulate a gesture like a hover, we only have *touch* gesture for this purpose. Last PR, we added an icon (info icon) if the user wants to read more about some location the user clicks it.

Again, It's only one suggestion **(no pressure to do it 😄)**. I think it could be better if you can change your hover over the title (and underline) to an icon.

Reply:
It's not final, we could easily switch this to a link  ("More", "See details")

Reply:
Just added the link for a better UX😄